src/util/scopedoverridecursor.h

```diff
@@ -1,16 +1,16 @@
 #pragma once
 
-#include <QApplication>
+#include <QGuiApplication>
 
 class ScopedOverrideCursor {
   public:
     inline explicit ScopedOverrideCursor(const QCursor& cursor) {
-        QApplication::setOverrideCursor(cursor);
-        QApplication::processEvents();
+        QGuiApplication::setOverrideCursor(cursor);
+        QGuiApplication::processEvents();
```

Comment:
QCoreApplication::processEvents()?

There should be a compiler or linter flag for detecting these kinds of inconsistencies.

Reply:
I have added this to the last commit.